src/blocks/net.rs

```diff
@@ -699,6 +686,35 @@ impl ConfigBlock for Net {
     }
 }
 
+fn old_format(net_config: &toml::Value) -> Option<String> {
+    // List of decprecated options
+    let old_strings = [
+        "ssid",
+        "signal_strength",
+        "signal_strength_bar",
+        "bitrate",
+        "ip",
+        "ipv6",
+        "speed_up",
+        "speed_down",
+        "graph_up",
+        "graph_down",
+    ];
+
+    let mut old_format = Vec::new();
```

Comment:
In the old config `speed_up` and `speed_down` defaulted to `true`, so we we need to change the logic here to account for that.

i.e.
In 0.13 this would show `ip`, `speed_up` and `speed_down`.
```
[[block]]
block = "net"
ip = true
```

In 0.13 this would show just `ip`.
```
[[block]]
block = "net"
ip = true
speed_down = false
speed_up = false
```

Reply:
Should be resolved in this commit: https://github.com/greshake/i3status-rust/pull/770/commits/8dbc8f512159ec5f9633fedac3698c6e8cb6d986